services/user.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User

```

Comment:
The `User` model is imported but not used in the file. Consider removing this import to clean up the code.